src/shims/unix/linux/foreign_items.rs

```diff
@@ -217,6 +218,14 @@ pub trait EvalContextExt<'mir, 'tcx: 'mir>: crate::MiriInterpCxExt<'mir, 'tcx> {
                 this.write_null(dest)?;
             }
 
+            "mmap64" => {
```

Comment:
You put this into the category of
```
// Incomplete shims that we "stub out" just to get pre-main initialization code to work.
// These shims are enabled only when the caller is in the standard library.
```
That doesn't seem right.

Reply:
I didn't notice that comment :-). I've now moved it to "File related shims".